packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -38,6 +39,23 @@ const defaultProps = {
   isAdditional: false
 };
 
+function createECMItemComponents(activity, props) {
+  const links = [];
+
+  for (const item of activity.links.items) {
```

Comment:
Let's verify that `links` and `links.items` exist so it doesn't crash the widget if the data changes on us again.